posthog/api/team.py

```diff
@@ -75,6 +75,8 @@ class CachingTeamSerializer(serializers.ModelSerializer):
     Has all parameters needed for a successful decide request.
     """
 
+    session_recording_sample_rate = serializers.DecimalField(min_value=0, max_value=1, max_digits=3, decimal_places=2)
```

Comment:
random curiosity: do you need this here when you have it defined on the model already?

Reply:
You don't 🙈 I was just following my nose earlier. Serializer has the benefit you don't hit the DB, Model has the benefit the validation applies if you don't go through the serializer. On balance I think the second is more important